warehouse/integrations/azure-synapse/azure-synapse.go

```diff
@@ -173,7 +173,7 @@ func (as *AzureSynapse) getConnectionCredentials() credentials {
 func columnsWithDataTypes(columns model.TableSchema, prefix string) string {
 	var arr []string
 	for name, dataType := range columns {
-		arr = append(arr, fmt.Sprintf(`%s%s %s`, prefix, name, rudderDataTypesMapToMssql[dataType]))
+		arr = append(arr, fmt.Sprintf(`"%s%s" %s`, prefix, name, rudderDataTypesMapToMssql[dataType]))
```

Comment:
This got missed when we added support for skipping escaping keywords. https://github.com/rudderlabs/rudder-server/pull/2115